cmake/modules/AddSwiftHostLibrary.cmake

```diff
@@ -77,8 +77,28 @@ function(add_swift_syntax_library name)
         -emit-module-path;${module_file};
         -emit-module-source-info-path;${module_sourceinfo_file};
         -emit-module-interface-path;${module_interface_file};
-        -emit-private-module-interface-path;${module_private_interface_file}
+        -emit-private-module-interface-path;${module_private_interface_file};
+        -no-verify-emitted-module-interface
```

Comment:
Without this, the driver invokes `-frontend -typecheck-module-from-interface` job and that emits confusing warnings:
```
<unknown>:0: warning: ignoring -experimental-allow-non-resilient-access (overriden by -compile-module-from-interface or -typecheck-module-from-interface)
<unknown>:0: warning: ignoring -experimental-package-cmo (requires -experimental-allow-non-resilient-access)
```

Reply:
If they are just warnings, is it necessary to opt out of interface verification? We should just fix the diagnostics and I don't want you to get stuck with a configuration where the module interface is not verified (new problems with generated interface could escape as a result).

Reply:
OK, it's not really necessary to disable.